tests/sentry/grouping/snapshots/test_grouphash_metadata/test_metadata_from_variants/newstyle@2023_01_11/unreal_event_capture_mac.pysnap

```diff
@@ -24,96 +24,181 @@ metrics with tags: {
 ---
 contributing variants:
   app*
-    hash: "9e04decaf79ecba9dc0314dc0edd3993"
+    hash: "54e8028fb2526cf31b12dd66c01ad9e2"
```

Comment:
This error already had an `app` hash because of `family:native package:/Users/** +app`.